### PR TITLE
Replace Links

### PR DESCRIPTION
**Pull request by Tishasoumya-02**
Replace links  https://training.plone.org/5/volto/index.html with https://training.plone.org/volto/index.html

### CODE REVIEW

**Comment by sudhanshu1309** [comment]
Pull requests must include a towncrier news item. This is a file that is placed in the root of the repository directory at /news.
https://6.docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request

File: news/#4511.bug [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix urls from .plone.org/5/volto/ to .plone.org/volto/ @Tishasoumya-02
```
[the remark this answers is not in the document]
Comment (Tishasoumya-02) [reply news/#4511.bug]: Yes @stevepiercy

**Comment by stevepiercy** [comment]
Let's wait to see the response from the Volto team regarding [@ksuess's question](https://github.com/plone/training/issues/672#issuecomment-1467426883), as we are not certain about the status of the Volto training.

File: news/4511.documentation [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix URLs from `training.plone.org/5/volto/` to `training.plone.org/volto/` @Tishasoumya-02
```
Comment (stevepiercy) [inline news/4511.documentation:1]:
```suggestion
Fix URLs from `training.plone.org/5/volto/` to `2022.training.plone.org/volto/` @Tishasoumya-02
```

Comment (stevepiercy) [reply news/4511.documentation:1]: I forgot to update this as well.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: packages/generator-volto/generators/app/templates/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Documentation
 
-A training on how to create your own website using Volto is available as part of the Plone training at [https://training.plone.org/5/volto/index.html](https://training.plone.org/5/volto/index.html).
+A training on how to create your own website using Volto is available as part of the Plone training at [https://training.plone.org/volto/index.html](https://training.plone.org/volto/index.html).
```
[the remark this answers is not in the document]
Comment (Tishasoumya-02) [reply packages/generator-volto/generators/app/templates/README.md:3]: I don't know if it's only me but this https://2022.training.plone.org/volto/index.html  isn't working for me.

[the remark this answers is not in the document]
Comment (stevepiercy) [reply packages/generator-volto/generators/app/templates/README.md:3]: It does not yet, but it is in the plan. See https://github.com/plone/training/issues/672.

We will hold merging this PR until we set up the new hostname.

[the remark this answers is not in the document]
Comment (Tishasoumya-02) [reply packages/generator-volto/generators/app/templates/README.md:3]: Okay Understood,will make the changes

**Comment by stevepiercy** [comment]
Closing as duplicate of https://github.com/plone/volto/pull/4523

**Comment by stevepiercy** [comment]
Apologies @Tishasoumya-02, I did not realize that you and @yahya-cloud were both working on the same issue and duplicating efforts, as I am handling a flood of reviews. To prevent this in the future, it is best practice to:

- Create an issue to describe the problem
- Assign yourself to the issue, claiming it
- Submit a pull request, including the following:
  - Include "Closes #IssueNumber" in a GitHub comment to automatically close the original issue number.
  - Include the issue number in the PR's title for easy reference.
  - Include the issue number being closed in the news item.

I will also try to pay closer attention to what is actually being worked on. Thank you for both of your collaboration and contributions to Plone!

**Pull request closed without merging**